jsx/src/components/DynamicTable/DynamicTable.jsx

```diff
@@ -0,0 +1,244 @@
+import React, { useState } from "react";
+import "./table-select.css";
+
+export default class DynamicTable extends React.Component {
```

Comment:
Hello @naatebarber, I am working on making DynamicTable functional, however I am not sure how to convert the following renderKeyRows class to a function: 

![image](https://user-images.githubusercontent.com/58890859/189358316-6bef8c8a-7821-4441-94b5-a174cf5ed31d.png)

The problem here is that I can no longer use `var context = this;` in here because it can only be used in classes and without it, it leads to an endless loop when rendering. Is there any way to do something similar for functions? Many thanks!